deps/wazuh_testing/setup.py

```diff
@@ -60,7 +60,7 @@ def get_files_from_directory(directory):
 
 setup(
     name='wazuh_testing',
-    version='4.4.0',
+    version='4.4.2',
```

Comment:
Would it be possible to get this parametrized? So that it is managed with the version of the repository so it is not hardcoded each time?